pmm-app/src/shared/components/Elements/Overlay/Overlay.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { FC } from 'react';
+import { Spinner } from '@grafana/ui';
+import { cx } from 'emotion';
+import { OverlayProps } from './Overlay.types';
+import { styles } from './Overlay.styles';
+
+export const Overlay: FC<OverlayProps> = ({
+  children,
+  className,
+  dataQa,
+  isPending,
+  size = 20
+}) => (
+  <div className={cx(styles.overlayWrapper, className)}>
+    {isPending
+      ? (
+        <>
+          <div className={styles.overlay} data-qa={dataQa}>
```

Comment:
Let's remove data-qa from here in favour of making one just for the div below - it's more important to test the children than the spinner.